src/main/resources/TrialUser/userPage.html

```diff
@@ -38,16 +38,32 @@ <h1 class="jumbotron-heading" >Welcome</h1> <br>
                     <td  th:text="${user.userId}"><span> ID </span></td>
                     <td  th:text="${user.username}"><span> Username </span></td>
                     <td  th:text="${user.role}"><span> Role </span></td>
-                    <td  th:text="${user.apiCallCount}"><span> Call Count </span></td>
-                    <td  th:text="${user.apiCallLimit}"><span> Call Limit </span></td>
+                    <td  id="count" th:text="${user.apiCallCount}"><span> Call Count </span></td>
+                    <td  id="limit" th:text="${user.apiCallLimit}"><span> Call Limit </span></td>
                 </tr>
                 </tbody>
             </table>
+
             <form th:action="@{/user/makeAPICall}" method="post">
-                <input class="btn btn-primary btn-lg" type="submit" value="Call API">
+                <input class="btn btn-primary btn-lg" type="submit" id="apibutton" value="Call API">
                 <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
 
             </form>
+            <script type="text/javascript">
+                window.onload = function() {
+                    function createAlert() {
+                        var count = document.getElementById("count").innerText;
+                        console.log(count);
+                        var limit = document.getElementById("limit").innerText;
+                        console.log(limit);
+
+                        if (count === limit) {
+                            alert("You have reached your limit. No new calls can be made!");
+                        }
+                    }
+                    document.getElementById("apibutton").addEventListener("click", createAlert);
+                }
+            </script>
```

Comment:
is this the best way to do this?

Reply:
Technically it could be moved to a js file, but since this is the only js this page will probably ever see, its fine for now.